examples/DancingGoat/AdminComponents/Apps/SampleDataGenerator/SampleDataGeneratorApplication.cs

```diff
@@ -1,8 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
-
-using CMS.Base;
+﻿using CMS.Base;
```

Comment:
I know it's out of the scope of this task, but can you please fix all warnings as well? It doesn't seem like a lot of work.